react-forms/src/DateTimeUI/DateTimePopup.tsx

```diff
@@ -63,45 +63,46 @@ const WrapperDiv = styled.div<IWrapperProps>`
   }`
 
 interface IProps {
-  DateTime: moment.Moment
-  Setter: (record: moment.Moment) => void;
-  Valid: boolean;
-  Feedback?: string;
-  Type: ('datetime' | 'date' | 'time' );
-  Show: boolean,
-  Top: number,
-  Center: number,
+    DateTime: moment.Moment
+    Setter: (record: moment.Moment) => void;
+    Valid: boolean;
+    Feedback?: string;
+    Type: ('datetime' | 'date' | 'time');
```

Comment:
it may be worth changing that to `datetime-local` and use the same type as above